### DISCUSSION TITLE
Arduino Nano ESP32

### DISCUSSION THREAD

**Discussion by buck2825** [MobiFlight/MobiFlight-FirmwareSource#321] [Q&A]
Is the Nano ESP32 compatible with MobiFlight?  can it be?

**Comment by elral** [comment]
No, the Nano ESP32 is not supported.

You could compile your own firmware for this board by adapting the platformio.ini. But you have also to upload the FW by yourself as also the connector does not support this board.

As an example for the platformio.ini see [here](https://github.com/elral/MobiFlight-FirmwareSource/blob/Nano_ESP32/platformio.ini). When I tested it there were problems with the servo library. That's the reason why the servo support is deactivated.
I don't know if this is solved in the meantime.
